### PR TITLE
Allow plugin models to be reported in the model count metrics.

### PR DESCRIPTION
**Pull request by Kircheneer**
This allows the models reported in the model count metrics to not only come from nautobot core, but also from plugins. It utilizes a `_module` key in the configuration, which defaults to `nautobot` both for backwards compatibility and ease of use.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -112,7 +112,7 @@ The behavior of the app_metrics feature can be controlled with the following lis
 - `gitrepositories` boolean (default **False**), publish stats about the gitrepositories (success, warning, info, failure)
 - `jobs` boolean (default **False**), publish stats about the jobs (success, warning, info, failure)
 - `queues` boolean (default **False**), publish stats about RQ Worker (nbr of worker, nbr and type of job in the different queues)
-- `models` nested dict, publish the count for a given object (Nbr Device, Nbr IP etc.. ). The first level must be the name of the module in lowercase (dcim, ipam etc..), the second level must be the name of the object (usually starting with a uppercase)
+- `models` nested dict, publish the count for a given object (Nbr Device, Nbr IP etc.. ). The first level must be the name of the module in lowercase (dcim, ipam etc..), the second level must be the name of the object (usually starting with a uppercase). Note that you can include models from plugins my specifying the module name under the "_module" key
```
Comment (glennmatthews) [inline docs/index.md:115]:
```suggestion
- `models` nested dict, publish the count for a given object (Nbr Device, Nbr IP etc.. ). The first level must be the name of the module in lowercase (dcim, ipam etc..), the second level must be the name of the object (usually starting with a uppercase). Note that you can include models from plugins by specifying the module name under the "_module" key
```

File: pyproject.toml [hunk 2]
```diff
@@ -19,6 +19,7 @@ packages = [
 [tool.poetry.dependencies]
 python = "^3.7"
 nautobot = "^1.2.0"
+setuptools = "^65.6.3"
```
Comment (glennmatthews) [inline pyproject.toml:22]: Is this a temporary workaround for an issue in newer setuptools versions? Will we want to remove it soon? Does it conflict with any version of Nautobot's dependencies?

**Comment by jedelman8** [comment]
@davidban77 - since this was approved, can we get this merged?

**Comment by davidban77** [comment]
@jedelman8 it needs some conflict-resolving but will sync with @Kircheneer

**Comment by Kircheneer** [comment]
Done with #68

**Pull request closed without merging**